src/components/Navigation/Navigation.jsx

```diff
@@ -15,13 +31,13 @@ return (
         </div>
         <ul className="nav-bar-links nav-section-items">
             <li className="list-inline-item">
-                <Link className="link" href="/trending">Trending</Link>
+                <a className="link" href="/trending">Trending</a>
             </li>
             <li className="list-inline-item">
-                <Link className="link" href="/new">New</Link>
+                <a className="link" href="/new">New</a>
             </li>
             <li className="list-inline-item">
-                <Link className="link" href="/grooming">For You</Link>
+                <a className="link" href="/grooming">For You</a>
```

Comment:
suggestion :you should use Link instead of anchor tag to avoid server loading.LInk tag will take "to" attribut for route link,earlier in code (which you have deleted )you used "href" in LInk.In